src/commands/openYAMLConfigFile.ts

```diff
@@ -23,7 +23,14 @@ export async function openYAMLConfigFile(context: IActionContext, node?: StaticW
     if (node instanceof StaticWebAppTreeItem || node instanceof EnvironmentTreeItem && node.gitHubConfigGroupTreeItems.length === 0) {
         const defaultHostname: string = node instanceof StaticWebAppTreeItem ? node.defaultHostname : node.parent.defaultHostname;
         const ymlFileName: string = `.github/workflows/azure-static-web-apps-${defaultHostname.split('.')[0]}.yml`;
-        return await openUrl(`${node.repositoryUrl}/edit/${node.branch}/${ymlFileName}`);
+        const localYamlFiles: Uri[] = await workspace.findFiles(ymlFileName);
+
+        if (localYamlFiles.length === 1) {
+            await window.showTextDocument(localYamlFiles[0]);
```

Comment:
I think that you could combine this with the logic below where you prompt the user for a yaml file.  It's kind of weird, but you have two different branches to open a local yaml file now.